tests/components/modbus/test_binary_sensor.py

```diff
@@ -341,31 +343,31 @@ async def test_config_slave_binary_sensor(hass, mock_modbus):
     "config_addon,register_words,expected, slaves",
     [
         (
-            {CONF_SLAVE_COUNT: 1},
+            {CONF_SLAVE_COUNT: 1, CONF_UNIQUE_ID: "ground_floor_sensor"},
```

Comment:
ground_floor_sensor should be a constant (top of this file) to avoid confusion if it changed in one of the instances.